scripts/bundleSize/bundleSize.test.js

```diff
@@ -1,30 +1,214 @@
+        'modern.shared-2222.js',
+        'modern.shared-3333.js',
+      ];
+    }
+    if (pageType === 'OnDemandAudioPage') {
+      return [
+        'modern.OnDemandAudioPage-31ecd969.ec6af2d0.js',
+        'modern.commons-1111.js',
+        'modern.commons-2222.js',
+        'modern.1111-lib-1111.js',
+        'modern.shared-2222.js',
+        'modern.shared-3333.js',
+      ];
+    }
+    if (pageType === 'OnDemandTvPage') {
+      return [
+        'modern.OnDemandTvPage-31ecd969.de41ab7f.js',
+        'modern.commons-1111.js',
+        'modern.1111-lib-1111.js',
+        'modern.3333-lib-2222.js',
+        'modern.shared-1111.js',
+      ];
+    }
+    if (pageType === 'PhotoGalleryPage') {
+      return [
+        'modern.PhotoGalleryPage-e94df663.a733283a.js',
+        'modern.commons-1111.js',
+        'modern.commons-2222.js',
+        'modern.1111-lib-1111.js',
+        'modern.shared-1111.js',
+        'modern.shared-3333.js',
+      ];
+    }
+    if (pageType === 'ErrorPage') {
+      return [
+        'modern.ErrorPage-31ecd969.31473c35.js',
+        'modern.commons-1111.js',
+        'modern.commons-2222.js',
+        'modern.1111-lib-1111.js',
+        'modern.shared-1111.js',
+        'modern.shared-2222.js',
+        'modern.shared-3333.js',
+      ];
+    }
+    if (pageType === 'IdxPage') {
+      return [
+        'modern.IdxPage-31ecd969.31473c35.js',
+        'modern.commons-1111.js',
+        'modern.commons-2222.js',
+        'modern.1111-lib-1111.js',
+        'modern.3333-lib-2222.js',
+        'modern.shared-1111.js',
+      ];
+    }
+    if (pageType === 'FeatureIdxPage') {
+      return [
+        'modern.FeatureIdxPage-31ecd969.31473c35.js',
+        'modern.commons-1111.js',
+        'modern.commons-2222.js',
+        'modern.1111-lib-1111.js',
+        'modern.3333-lib-2222.js',
+        'modern.shared-1111.js',
+      ];
+    }
+
+    throw Error(`page type ${pageType} not mocked`);
+  },
 }));
-jest.mock('fs');
-jest.mock('./bundleSizeConfig', () => ({
+
+jest.unstable_mockModule('./bundleSizeConfig', () => ({
   MIN_SIZE: 632,
   MAX_SIZE: 728,
 }));
 
+jest.unstable_mockModule(
+  './serviceList.js',
+  () => ({ default: ["service1", "service2"] }),
+);
+
+jest.unstable_mockModule('fs', () => ({
+  default: {
+    readdirSync: jest.fn(),
+    statSync: jest.fn(),
+  },
+}));
+
+const start = jest.fn();
+const succeed = jest.fn();
+const fail = jest.fn();
+
+jest.unstable_mockModule('ora', () => {
+  const mock = jest.fn();
+  mock.mockReturnValue({
+    start,
+    succeed,
+    fail,
+  });
+  return {
+    default: mock,
+  };
+});
+
+jest.unstable_mockModule('chalk', () => ({
+  default: {
+    red: a => a,
+    green: a => a,
+    blue: a => a,
+    yellow: a => a,
+    cyan: { bold: b => b },
+    bold: a => a,
+  },
+}));
+
 const setUpFSMocks = (service1FileSize, service2FileSize) => {
-  beforeEach(() => {
+  beforeEach(async () => {
+    const fs = await import('fs');
```

Comment:
Can we do this, so that we don't have to specify `fs.default` below?
```suggestion
    const { default: fs } = await import('fs');
```